reconstruction/fmt/src/main/java/org/jlab/service/fmt/FMTEngine.java

```diff
@@ -50,7 +50,10 @@ public boolean init() {
         String[] tables = new String[]{
             "/geometry/beam/position",
             "/calibration/mvt/fmt_time",
-            "/calibration/mvt/fmt_status"
+            "/calibration/mvt/fmt_status",
+            "/calibration/mvt/fmt_voltage_drift",
+            "/calibration/mvt/fmt_voltage_strip_inout",
```

Comment:
is this table used? is it intended for the numbers mentioned above?

Reply:
Yes, this table is used to get the voltages in the part of the code that reads hits.
The numbers above correspond to the strip values defining the boundaries of the sectors.  Not related to the threshold implementation.  But I moved them to the constants class.